disbatch/disB_submit.sh

```diff
@@ -1,6 +1,8 @@
 module load disBatch/beta
-mkdir ./ceph_yliu_data/log
-mkdir ./ceph_yliu_data/results
-sbatch -p scc -N 3 -c 1 --mincpus=100 disBatch -p ./ceph_yliu_data/log disB_tasks_all
+
+DIR_LOG=./log
+mkdir -p $DIR_LOG
+
+sbatch -p scc -N 3 -c 1 --mincpus=100 disBatch -p $DIR_LOG disB_tasks_all
 #./ceph_yliu_data/log/disB_tasks_disBatch_220922145716_553_dbUtil.sh --mon
```

Comment:
Should remove this lingering comment

Reply:
Add comments. 
#use the following commands to start monitoring interface
#replace [ids] with real ones
#./log/disB_tasks_disBatch_[ids]_dbUtil.sh --mon